src/RestfulServer.php

```diff
@@ -331,15 +331,15 @@ protected function getDataFormatter($includeAcceptHeader = false, $className = n
         // get formatter
         if (!empty($extension)) {
             $formatter = DataFormatter::for_extension($extension);
-        } elseif ($includeAcceptHeader && !empty($accept) && $accept != '*/*') {
+        } elseif ($includeAcceptHeader && !empty($accept) && strpos($accept, '*/*') === false) {
```

Comment:
This is a different change from the other two lines. Would you mind either explaining this change to show the different type of input data you've got for `$accept` that justifies this change? A unit test would be ideal, but not required.

The config changes are good :)

Reply:
My accept value is "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", which disables default_extension, because it is != "*/*", while IMO it should not, because it contains "*/*" which is just as good.